internal/repository/postgres.go

```diff
@@ -133,7 +133,8 @@ func (s *RepoPostgres) Setup(ctx context.Context) {
 			nodes_util_list JSONB,
 			UNIQUE (id),
 			PRIMARY KEY (id))`,
-		// for yunikorn-core/pkg/webservice/dao/ContainerHistoryDAOInfo and yunikorn-core/pkg/webservice/dao/ApplicationHistoryDAOInfo
+		// for yunikorn-core/pkg/webservice/dao/ContainerHistoryDAOInfo and
```

Comment:
do we need these comments?

Reply:
Not really needed, the yunikorn-core structs are referenced in the postgres repo layer, in the SQL upsert logic.
Removed in commit 7927a9d